dev_tools/utils/nxdl_utils.py

```diff
@@ -260,59 +259,35 @@ def belongs_to_capital(params):
 
 def get_local_name_from_xml(element):
     """Helper function to extract the element tag without the namespace."""
-    return element.tag[element.tag.rindex("}") + 1 :]
+    return remove_namespace_from_tag(element.tag)
```

Comment:
RHS preferred, reusing given function however the remove_namespace_from_tag function in line 16 makes an additional lead in check which the LHS single-line expression does not